### PR TITLE
recipe project: edit copy, include code examples

### PR DESCRIPTION
**Pull request by sheltonbcode**
The changes I've proposed are obviously not do or die, but I think some of the copy throughout the Foundations curriculum could be refreshed/cleaned up a bit and I'm happy to go through and help polish some of that copy.

In this case I sought to bring clarity to some of the recipe project instructions and introduce more example code to help learners with more visual example dependency.

**Happy to be here and I'm open to any and all feedback!**

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

File: foundations/html_css/html-foundations/project-recipes.md [hunk 1]
```diff
@@ -48,11 +48,11 @@ Okay, that's enough Git for the moment -- time to actually build stuff!
 1. Create a new directory within the `odin-recipes` directory and name it `recipes`.
 2. Create a new HTML file within the  `recipes` directory and name it after the recipe it will contain. For example `lasagna.html`. You can use the name of your favorite dish or, if you need some inspiration, you can [find a recipe to use here](https://www.allrecipes.com/).
 3. For now, just include an `h1` heading with the recipe's name as its content.
-4. Back in the `index.html` file, add a link to the recipe page you just created. The text of the link should again be the recipe name.
+4. Back in the `index.html` file, add a link to the recipe page you just created. Example: Under the <h1>Odin Recipes</h1> heading, write out the link like so: <a href="recipename.html">Recipe Title</a>. The text of the link should again be the recipe name.
```
Comment (zachmmeyer) [inline foundations/html_css/html-foundations/project-recipes.md:51]:
```suggestion
4. Back in the `index.html` file, add a link to the recipe page you just created. Example: Under the `<h1>Odin Recipes</h1>` heading, write out the link like so: `<a href="recipename.html">Recipe Title</a>`. The text of the link should again be the recipe name.
```
Without backticks, your code examples will get parsed into actual page elements.
Please do next time try out your additions with the Markdown Preview tool. That's why it's listed as a requirement.

File: foundations/html_css/html-foundations/project-recipes.md [hunk 2]
```diff
@@ -62,7 +62,16 @@ The recipe page should have the following content:
...
 1. Add two more recipes with identical page structures to the recipe page you've already created.
-2. Don't forget to link to the new recipes on the index page. Also, consider putting all the links in an unordered list so they aren't all on one line.
+2. Don't forget to link to the new recipes on the index page. Also, consider putting all the links in an unordered list so they aren't all on one line. 
+
+Example: 
+  <ul>
+    <li><a href="yourrecipe.html">Recipe Title 1</a>.</li>
+    <li><a href="yourrecipe.html">Recipe Title 2</a></li>
+    <li><a href="yourrecipe.html">Recipe Title 3</a></li>
+  </ul>
```
Comment (zachmmeyer) [inline foundations/html_css/html-foundations/project-recipes.md:72]:
```suggestion
~~~html 
 <ul>
    <li><a href="yourrecipe.html">Recipe Title 1</a>.</li>
    <li><a href="yourrecipe.html">Recipe Title 2</a></li>
    <li><a href="yourrecipe.html">Recipe Title 3</a></li>
  </ul>
~~~
Same issue here. The layout style guide has information on adding code blocks.
```

**Review by zachmmeyer — CHANGES_REQUESTED** [review]
Thanks for contributing. I have two requested changes.

**Review by zachmmeyer — APPROVED** [review]

File: foundations/html_css/html-foundations/project-recipes.md (hunk 1 above)
Comment (sheltonbcode) [reply foundations/html_css/html-foundations/project-recipes.md]: Hey Zach thank you for your comment! I'll be sure to add those code blocks in the future!

**Comment by sheltonbcode** [comment]
Hey Zach, a quick note. I checked the live changes on the Recipes page and I've noticed that some of the approved changes included your comments on my suggestions. Also one of the instructions appears twice.

**Comment by zachmmeyer** [comment]
> Hey Zach, a quick note. I checked the live changes on the Recipes page and I've noticed that some of the approved changes included your comments on my suggestions. Also one of the instructions appears twice.

Thanks for that, fixed it!

**Pull request merged**